### PR TITLE
Error output in console

### LINKED ISSUE #160: Error output in console
## Current Behavior
`Failed to execute ‘postMessage’ on ‘DOMWindow’: The target origin provided (‘https://verify.walletconnect.com’) does not match the recipient window’s origin (‘https://panel.fs.neo.org’).`

## Expected Behavior
No errors.

## Steps to Reproduce
Open panel.fs.neo.org with existing session.

## Your Environment
* Version of the product used: 0.4.0
* Operating System and version (`uname -a`): Firefox 115.7.0esr

### PR DESCRIPTION
**Pull request by mike-petrov**
Signed-off-by: Mikhail Petrov <mike@nspcc.ru>

closes #160

### CODE REVIEW

**Comment by mike-petrov** [comment]
`Failed to execute ‘postMessage’ on ‘DOMWindow’: The target origin provided (‘https://verify.walletconnect.com’) does not match the recipient window’s origin (‘https://panel.fs.neo.org’)`

This error occurs because we're calling testInvoke in the function onNeoBalance before initialisation. Initialization is done by sending a socket (wss://relay.walletconnect.com/), so the interface doesn't have any triggers for this. I think timeout is the best option in this case.

File: src/Profile.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const Profile = ({
 
 	useEffect(() => {
 		if (walletData && walletData.account && !isLoading) {
-			onNeoBalance();
+			setTimeout(() => onNeoBalance(), 500);
```
Comment (roman-khimov) [inline src/Profile.js:73]: So if initialization takes longer it'll still fail?

Comment (mike-petrov) [reply src/Profile.js:73]: yes it's a synthetic timeout, but I checked on slow 3g, it was without error.

Comment (roman-khimov) [reply src/Profile.js:73]: Kludgy. But I don't have any better idea.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**